LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
@HadessdV @AlexanderBodin , guys be aware we already talked about BSD-License in the setup.py 

Just wanted to make sure we don't mix up licenses.

Reply:
ah, nevermind. I changed it to MIT here:
https://github.com/PaesslerAG/pytest-reference-formatter/blob/wbpvcm-add-travis-ci/setup.py